bin/scc

```diff
@@ -72,8 +75,16 @@ def make_commander
     atlas_interval: $opts[:"atlas-interval"].to_i
   }
 
-  StellarCoreCommander::Commander.new($opts[:"process"], opts).tap do |c|
-    c.cleanup_at_exit!
+  destination = $opts[:"destination"]
+  if destination
+    clean_up_destination = false
+  else
+    destination = Dir.mktmpdir("scc")
+    clean_up_destination = true
+  end
+
+  StellarCoreCommander::Commander.new($opts[:"process"], destination, opts).tap do |c|
+    c.cleanup_at_exit!(clean_up_destination)
```

Comment:
This is another one of those not-super-happy-with-it-but-probably-good-enough lines.

This has me thinking that the at_exit handling should be done at the shell (bin/scc) layer rather than in the commander. But since it's in the commander we'll push removal behavior down to it.